plugins/lookup/tss.py

```diff
@@ -136,7 +136,16 @@
 class LookupModule(LookupBase):
     @staticmethod
     def Client(server_parameters):
-        return SecretServer(**server_parameters)
+        authorizer = PasswordGrantAuthorizer(
+            server_parameters["base_url"],
+            server_parameters["username"],
+            server_parameters["password"],
+            server_parameters["token_path_uri"],
+        )
```

Comment:
```suggestion
        if LooseVersion(thycotic_version) < LooseVersion('1.0.0'):
            return SecretServer(**server_parameters)
            
        authorizer = PasswordGrantAuthorizer(
            server_parameters["base_url"],
            server_parameters["username"],
            server_parameters["password"],
            server_parameters["token_path_uri"],
        )
```

`token_path_uri` is not a valid argument in `0.0.6` so preserving the existing behavior for versions < `1.0.0` should give appropriate time for deprecation (thycotic_version = `thycotic.__version__`). See [here](https://github.com/thycotic/python-tss-sdk/blob/9727ffea263fd29ed7cb0a64325a95a735162365/thycotic/secrets/server.py#L187-L193)